web/app/js/components/TapEventTable.jsx

```diff
@@ -36,34 +36,31 @@ const genFilterOptionList = options => _.map(options,  (_v, k) => {
 });
 
 let tapColumns = (filterOptions, ResourceLink) => [
-  {
-    title: "ID",
-    dataIndex: "requestInit.http.requestInit.id.stream"
-  },
   {
     title: "Direction",
+    key: "direction",
     dataIndex: "base.proxyDirection",
+    width: "60px",
```

Comment:
It looks like 60px isn't quite wide enough for this column, causing the title text to wrap:

![image](https://user-images.githubusercontent.com/9226/45379990-651ee900-b5b6-11e8-9e56-0a77487b4476.png)